test/compilable/test15371.d

```diff
@@ -0,0 +1,15 @@
+/*
+TEST_OUTPUT:
+---
+---
+*/
+
+import imports.test15371;
+
+void main()
+{
+    A a;
+    static assert(__traits(compiles, __traits(hasMember, A, "a")));
```

Comment:
Why is this wrapped in a `__traits(compiles`? We can drop that.

Reply:
I simply copy pasted the test. Yes we can, done.

Reply:
Thanks! `__traits(compiles` is a bit of an anti-pattern in compilation as you don't get the real error message if you change/break DMD and the test fails.